keyboards/inett_studio/sqx/hotswap/hotswap.h

```diff
@@ -0,0 +1,25 @@
+/**
+  * @file hotswap.h
+  *
+  */
+#pragma once
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+// The first section contains all of the arguements
+// The second converts the arguments into a two-dimensional array
+#define LAYOUT_60_wkl_split_bs_rshift( \
```

Comment:
This PCB's layout matches the `60_tsangan_hhkb` community layout.

```suggestion
#define LAYOUT_60_tsangan_hhkb( \
```